apps/web/src/app/globals.css

```diff
@@ -48,6 +48,10 @@
     opacity: 0.5;
     transition: all 0.3s ease;
   }
+
+  .app-box-shadow {
```

Comment:
Please could we add this to our tailwind config? In general we should never apply global styles, or use them for "common styling" as it goes against the way Tailwind is intended to be used. Happy to go into more detail, send more info on this!

https://v3.tailwindcss.com/docs/box-shadow#using-custom-values